src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/controller/OrchestrationSessionsController.kt

```diff
@@ -111,8 +111,14 @@ class OrchestrationSessionsController(private val visitSchedulerService: VisitSc
     @RequestParam(value = "sessionRestriction", required = false)
     @Parameter(description = "Filter sessions by session restriction - OPEN or CLOSED, if prisoner has CLOSED it will use that", example = "CLOSED", required = false)
     sessionRestriction: SessionRestriction = SessionRestriction.OPEN,
+    @RequestParam(value = "visitors", required = true)
+    @Parameter(
+      description = "List of visitors who require visit sessions",
+      example = "4729510",
+    )
+    visitors: List<Long>,
```

Comment:
With this being required, are we breaking the contract for other services calling this endpoint?

Reply:
Will check this as PVB might be using this and they don't have visitors